src/ucp/api/ucp_def.h

```diff
@@ -302,6 +324,32 @@ typedef struct ucp_err_handler {
 
 
 /**
+ * @brief BSD socket address specification.
+ *
+ * This structure should be initialized in @ref ucp_ep_params_t
+ */
+typedef struct ucp_addr_sock_addr {
+    struct sockaddr   *addr;           /**< Address in the form of a sockaddr */
+    socklen_t         addrlen;         /**< Address length */
+} ucp_addr_sock_addr_t;
+
```

Comment:
@yosefe should we move this struct out of ucp so that uct will also be able to use it when binding? 
to ucs?